### ISSUE TITLE
Yandex Navigator for Android

### ISSUE THREAD

**Issue by chemerisuk** [dpa99c/phonegap-launch-navigator#52]
Thanks for the plugin! It would be nice to add support for Yandex Navigator for Android as well. They have a [spec](https://github.com/yandexmobile/yandexmapkit-android/wiki/%D0%98%D0%BD%D1%82%D0%B5%D0%B3%D1%80%D0%B0%D1%86%D0%B8%D1%8F-%D1%81-%D0%9C%D0%BE%D0%B1%D0%B8%D0%BB%D1%8C%D0%BD%D1%8B%D0%BC%D0%B8-%D0%AF%D0%BD%D0%B4%D0%B5%D0%BA%D1%81.%D0%9A%D0%B0%D1%80%D1%82%D0%B0%D0%BC%D0%B8) but it's in Russian. Below is a code example for creating a route with translated comments:

```js
    Intent intent = new Intent("ru.yandex.yandexnavi.action.BUILD_ROUTE_ON_MAP");
    intent.setPackage("ru.yandex.yandexnavi");

    PackageManager pm = getPackageManager();
    List<ResolveInfo> infos = pm.queryIntentActivities(intent, 0);

    // check if Yandex Navigator is installed
    if (infos == null || infos.size() == 0) {
        // If not, show the app page at Google Play
        intent = new Intent(Intent.ACTION_VIEW);
        intent.setData(Uri.parse("market://details?id=ru.yandex.yandexnavi"));
    } else {
        intent.putExtra("lat_from", 55.751802);
        intent.putExtra("lon_from", 37.586684);
        intent.putExtra("lat_to", 55.758192);
        intent.putExtra("lon_to", 37.642817);
    }

    startActivity(intent);
```

**Comment by dpa99c** [comment]
Thanks, I'll add it to my to-do list. v3 of the plugin should currently implicitly support Yandex Navigator on Android via the geo: URI scheme, but adding explicit support will also allow start location to be specified.

**Comment by dpa99c** [comment]
Implemented by 26b5cb08e448cf408bc85545d23e52273fefbbe2 in [v3.0.4](https://github.com/dpa99c/phonegap-launch-navigator/tree/3.0.4)
